sdk/runanywhere-kotlin/src/jvmAndroidMain/kotlin/com/runanywhere/sdk/public/extensions/RunAnywhere+VoiceAgent.jvmAndroid.kt

```
@@ -274,190 +273,195 @@ actual suspend fun RunAnywhere.setVoiceSystemPrompt(prompt: String) {
+
+                                // If continuous mode, reset for next turn
+                                if (config.continuousMode && processed) {
+                                    lastSpeechTime = 0L
+                                    // Continue collecting audio for next turn
+                                }
                             }
                         }
                     }
                 }
             }
+        } catch (e: kotlinx.coroutines.CancellationException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)